modules/postgresql/versions.tf

```diff
@@ -29,6 +29,10 @@ terraform {
       source  = "hashicorp/google"
       version = ">= 4.4.0, < 5.0"
     }
+    google-beta = {
+      source  = "hashicorp/google-beta"
+      version = ">= 4.4.0, < 5.0"
+    }
   }
 
   provider_meta "google" {
```

Comment:
Could we also add provider_meta for the beta provider?
https://github.com/terraform-google-modules/terraform-google-sql-db/blob/704be5d94eb2e321aabe2c704a6610d569e0bbe2/modules/mssql/versions.tf#L27-L29